integreat_cms/api_client_firebase/apps.py

```diff
@@ -0,0 +1,54 @@
+"""
+Configuration of PushNotificationSender API app
+"""
+import json
+import logging
+import requests
+
+
+
+from django.apps import apps, AppConfig
+from django.conf import settings
+
+logger = logging.getLogger(__name__)
+
+
+class PushNotificationSenderApiConfig(AppConfig):
```

Comment:
```suggestion
class FirebaseApiConfig(AppConfig):
```
Our internal team vote has shown that we have a preference to keep the current naming scheme of `<service>_api`, so could you rename the directory to `firebase_api`? Thanks!